app/views/govuk_publishing_components/components/_select.html.erb

```diff
@@ -10,12 +10,23 @@
   heading_size = false unless shared_helper.valid_heading_size?(heading_size)
   error_message ||= nil
   error_id ||= nil
+  hint_text ||= nil
```

Comment:
I seems like we use `hint` for the hint text, looking at other component's APIs. I would stick to `hint` for consistency, although I appreciate `hint_text` might be clearer.